tests/cn/leaky_main.runtime.c

```diff
@@ -0,0 +1,29 @@
+// TODO - revisit this Dhruv over-confident change
+
+#ifndef CN_UTILS
+void *cn_malloc(unsigned long long);
+void cn_free_sized(void*, unsigned long long);
+#endif
+
+unsigned int *mkref(unsigned int x)
+/*@ trusted;
+requires
+    true;
+ensures
+    take v1_ = Owned(return);
+    v1_ == x;
+@*/
+{
+  unsigned int *p = cn_malloc(sizeof(unsigned int));
+  *p = x;
+  return p;
+}
+
+int main()
+/*@ trusted; @*/
+{
+    unsigned int *p = mkref(5);
+    /*@ assert (*p == 5u32); @*/
+    // TODO - consider decrementing stack-depth for main too?
```

Comment:
Can consider, although right now we are treating `main` as slightly 'special' - the addresses of C globals currently are added to the ghost state here since the stack depth is assumed to be 0, although I suppose that's not a huge change if we decide to add the stack depth increment and decrement here too